### PR TITLE
added rake task to update pingdom ips in a given YAML file

### PR DESCRIPTION
**Pull request by aldavidson**
see the task in Rakefile, or run "bundle exec rake -D", for full details

PVB will use this to update a set of firewall 'allow' rules, to allow Pingdom to monitor a sensu server in skyscape

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+end
+
+def file_or_stdout(filepath)
+  filepath.gsub(/\s+/,'').empty? ? STDOUT : File.open(filepath, 'wb+')
+end
+
+def output(filepath, string)
+  out = file_or_stdout(filepath)
+  out << string
+  out.close if out.is_a?(File)
```
Comment (SteveMarshall) [inline Rakefile]: I *think* you should just be OK to close `STDOUT` anyway, maybe?

File: Rakefile [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  def ip_list_rss_url
+    'https://www.pingdom.com/rss/probe_servers.xml'
+  end
+
+  def probe_ips
+    ips = []
+    parse_feed.elements.each('rss/channel/item/pingdom:ip'){ |el| ips << el.text }
+    ips
+  end
```
Comment (SteveMarshall) [inline Rakefile]: Could this be a `map` and allow us to dump the first and last lines of this?

**Comment by SteveMarshall** [comment]
Can you terminate `Gemfile` and `Rakefile` with a newline?

**Comment by aldavidson** [comment]
@SteveMarshall done

**Comment by SteveMarshall** [comment]
@aldavidson :boom:

**Pull request merged**